src/profile_wizard.cpp

```diff
@@ -875,26 +902,37 @@ struct AutoConnectCamera
 {
     GuideCamera *m_camera;
 
-    AutoConnectCamera(wxWindow *parent, const wxString& selection)
+    AutoConnectCamera(ProfileWizard *parent, const wxString& selection, bool forceSelection)
     {
+        wxString camDeviceId = GuideCamera::DEFAULT_CAMERA_ID;
         m_camera = GuideCamera::Factory(selection);
         pFrame->ClearAlert();
 
         if (m_camera)
         {
+            if (forceSelection)
+                camDeviceId = parent->ChooseCamDeviceId(m_camera);
+            else
+                camDeviceId = parent->GetCamDeviceId();
             wxBusyCursor busy;
-            m_camera->Connect(GuideCamera::DEFAULT_CAMERA_ID);
+            m_camera->Connect(camDeviceId);
             pFrame->ClearAlert();
         }
 
-        if (!m_camera || !m_camera->Connected)
+        if (m_camera && !m_camera->Connected)
         {
-            wxMessageBox(_("PHD2 could not connect to the camera so you may want to deal with that later. "
+            wxString preamble = (m_camera->CanSelectCamera() && parent->NumCamerasFound() == 0)
+                ? _("No cameras of that type were found")
+                : _("PHD2 cound not connect to the camera");
+
+            wxMessageBox(preamble +
+                         _(" so you may want to deal with that later. "
                            "In the meantime, you can just enter the pixel-size manually along with the "
                            "focal length and binning levels."));

```

Comment:
It will be better for translators if you make two separate strings rather than a concatenation of preamble + rest. When translating to another language, it may be more natural to express it some other way than preamble + " so ...".